ElectronClient/app/compile-package-info.js

```diff
@@ -16,6 +17,25 @@ const appId = packageInfo.build.appId;
 delete packageInfo.build;
 packageInfo.build = { appId: appId };
 
+let branch;
+let hash;
+let tag;
+try {
+	branch = execSync('git branch --show-current').toString().trim();
+	hash   = execSync('git log --pretty="%h" -1').toString().trim();
+	tag    = execSync('git describe --tags --exact-match').toString().trim();
+}
+catch(err) {
+	// don't throw error, if git not installed or not in PATH
```

Comment:
Indeed no need to throw an error but please add a `console.warn("Could not get git info", err)` so that we know there's an issue.

Reply:
Sure, will do.